l10n_it_account_stamp_sale/models/sale_order.py

```diff
@@ -1,5 +1,5 @@
-# -*- coding: utf-8 -*-
 # Copyright 2019 Alex Comba - Agile Business Group
+# Copyright 2020 Sergio Corato <https://github.com/sergiocorato>
```

Comment:
come mai qui hai aggiunto il tuo Copyright?

Reply:
Non si mette di default ogni volta che si lavora su un modulo?

Reply:
Si mette quando vengono fatte delle modifiche che lo giustificano. Questo non mi pare il caso :wink:

Reply:
ah bon, mi hai tolto un dubbio, lo tolgo :D

Reply:
in contributors invece ci va?

Reply:
sì certo!